### ISSUE TITLE
[CI] failing git-init on kythe repo

### ISSUE THREAD

**Issue by fangism** [google/verible#584]
After fixing #556, the next issue is related to Kythe as a dependency.

See this log:
https://travis-ci.org/github/google/verible/jobs/742535455

```
INFO: Repository io_kythe instantiated at:
  /src/verible/verible-v0.0-759-g7941857/WORKSPACE:143:15: in <toplevel>
Repository rule git_repository defined at:
  /root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
ERROR: An error occurred during the fetch of repository 'io_kythe':
   Traceback (most recent call last):
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git.bzl", line 181, column 30, in _git_repository_implementation
		update = _clone_or_update(ctx)
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git.bzl", line 36, column 20, in _clone_or_update
		git_ = git_repo(ctx, directory)
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git_worker.bzl", line 91, column 12, in git_repo
		_update(ctx, git_repo)
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git_worker.bzl", line 101, column 9, in _update
		init(ctx, git_repo)
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git_worker.bzl", line 118, column 15, in init
		_error(ctx.name, cl, st.stderr)
	File "/root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/bazel_tools/tools/build_defs/repo/git_worker.bzl", line 190, column 9, in _error
		fail("error running '%s' while working with @%s:\n%s" % (command_text, name, stderr))
Error in fail: error running 'git init /root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/io_kythe' while working with @io_kythe:
src/main/tools/process-wrapper-legacy.cc:80: "execvp(git, ...)": No such file or directory
ERROR: no such package '@io_kythe//': error running 'git init /root/.cache/bazel/_bazel_root/2ccca2c4c14abc5021f2a7b18d0dec9b/external/io_kythe' while working with @io_kythe:
src/main/tools/process-wrapper-legacy.cc:80: "execvp(git, ...)": No such file or directory
INFO: Elapsed time: 7.206s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
The command '/bin/sh -c bazel build --workspace_status_command=bazel/build-version.sh -c opt //...' returned a non-zero code: 1
The command "./.github/travis/script.sh" exited with 1.
```

FYI @ivan444 .

**Comment by fangism** [comment]
Could this be a matter of making sure `git` is installed on the executor?  (Bazel is invoking git as a means of fetching a repo.)

**Comment by ivan444** [comment]
Most likely. Kythe is introduced as git_repo, not http_repo.

**Comment by fangism** [comment]
@kgugala @wtatarski Have some bandwidth to investigate?

**Comment by wtatarski** [comment]
@fangism I installed some lacking dependencies but now I see that some libs are outdated:
https://github.com/antmicro/verible/runs/1413572108?check_suite_focus=true
So I guess here is some work to do.

**Comment by fangism** [comment]
Other test scripts we have use bazel 3.7.0.

https://cs.opensource.google/search?q=BAZEL_VERSION&sq=&ss=verible%2Fverible

**Comment by ivan444** [comment]
Maybe a simpler approach would be to integrate Kythe in a different way. We
don't need all of it.
But I don't have a concrete proposal at the moment.

**Comment by mithro** [comment]
Doesn't #603 obsolete this issue?

**Comment by mithro** [comment]
IE - As #603 is merged, Travis-CI is no longer being run.

**Comment by fangism** [comment]
This same problem occurs on github action, so I'll update the title.

**Comment by fangism** [comment]
Example recent log:
https://github.com/antmicro/verible/runs/1411135012?check_suite_focus=true

**Comment by wtatarski** [comment]
Fixed in #608
